packages/xsnap/src/globals.d.ts

```diff
@@ -0,0 +1,5 @@
+declare var issueCommand: (msg: ArrayBuffer) => ArrayBuffer;
+
+namespace global {
+  declare var issueCommand: (msg: ArrayBuffer) => ArrayBuffer;
+}
```

Comment:
Please add missing terminal newline. Doesn't `yarn lint-fix` do that for us?

Reply:
I suppose our lint rules don't apply to `.d.ts` files. fixing.